src/test/java/uk/gov/hmcts/darts/arm/service/UnstructuredToArmBatchProcessorTest.java

```diff
@@ -136,12 +136,14 @@ void setUp() throws IOException {
 
     @Test
     void testDartsArmClientConfigInBatchQuery() {
```

Comment:
Since we've touched the test here, consider renaming according to the new naming standard.